R/environment.R

```diff
@@ -108,43 +108,41 @@ r_environment <- function(name, version = NULL,
 
   if (is.null(custom_docker_image)) {
     if (use_gpu) {
-      base_image_with_address <- paste0("mcr.microsoft.com/azureml/base-",
+      env$docker$base_image <- paste0("mcr.microsoft.com/azureml/base-",
```

Comment:
so setting of base images is not handled in the EMS side?

Reply:
no, EMS does not know which base image to use for CPU/GPU target.